### PR TITLE
BGE: Set Batch Options

### PR DESCRIPTION
**Pull request by randi274**
# Critical Changes

# Changes
- Allow user to edit processing settings as part of the batch configuration wizard.
# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by randi274** [comment]
**lurch: attach W-029604

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000007fPL5QAM" target="_blank">W-029604</a>

File: src/labels/CustomLabels.labels [hunk 1]
```diff
@@ -2481,6 +2481,41 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
...
+        <protected>true</protected>
+        <shortDescription>Used as field help text since help text is not upgradeable.</shortDescription>
+        <value>The number of records to process at a time in each batch.</value>
+    </labels>
+    <labels>
+        <fullName>bgeBatchTemplateDonationDateRangeHelp</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Used as field help text since help text is not upgradeable.</shortDescription>
+        <value>Enter the numer of days from the Donation Date to consider when looking for a matching Opportunity or Payment. NPSP will choose the matching Opp or Payment whose date falls within the number of days AND is closest to the Donation Date.</value>
```
Comment (lmeerkatz) [inline src/labels/CustomLabels.labels]: numer -> number

File: src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardController.js [hunk 2]
```diff
@@ -69,12 +74,9 @@
             }
             model.getTemplateMetadata().cancel();
         } else if (channel === 'save' || buttonClick === 'save') {
-            var isValid = model.getTemplateFields().getDefaultFieldValidity(component);
-            if (isValid) {
-                var fieldOptions = component.get('v.templateFieldOptions.fieldGroups');
-                model.getTemplateFields().updateTemplateFieldOptions(fieldOptions);
-                model.save();
-            }
+            //todo: add validation
```
[the remark this answers is not in the document]
Comment (randi274) [reply src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardController.js]: TBD! It wasn't in the ticket, and still needs some consideration as to what's the reasonable expectation.

[the remark this answers is not in the document]
Comment (lmeerkatz) [reply src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardController.js]: It looks like there was validation previously that's being removed here. What was that?

[the remark this answers is not in the document]
Comment (randi274) [reply src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardController.js]: Good question! That moved up into the 'next' block with step 4. Step 4 is the default value setting; Step 5 is when we call save on the batch options, and that doesn't have any validation associated with it yet.

File: src/labels/CustomLabels.labels (hunk 1 above)
Comment (randi274) [reply src/labels/CustomLabels.labels]: Fixed, thanks!

**Comment by cpose** [comment]
QA Approved, Merging.

**Pull request merged**